src/containers/pages/JurisdictionAssignment/JurisdictionTable/index.tsx

```diff
@@ -67,18 +75,18 @@ export interface JurisdictionSelectorTableProps {
   tree?: TreeNode;
   plan: PlanDefinition;
   rootJurisdictionId: string;
+  getJurisdictionsMetadata: Dictionary<Dictionary<Dictionary<Meta>>>;
   currentParentId?: string;
   jurisdictionsMetadata: JurisdictionsMetadata[];
   serviceClass: typeof OpenSRPService;
   treeFetchedCreator: typeof fetchTree;
   currentParentNode?: TreeNode;
   currentChildren: TreeNode[];
-  selectNodeCreator: typeof selectNode;
-  deselectNodeCreator: typeof deselectNode;
-  autoSelectNodesCreator: typeof autoSelectNodes;
-  fetchPlanCreator: typeof addPlanDefinition;
+  selectNodeCreator: ActionCreator<SelectNodeAction>;
+  deselectNodeCreator: ActionCreator<DeselectNodeAction>;
+  autoSelectNodesCreator: ActionCreator<AutoSelectNodesAction>;
+  fetchPlanCreator: ActionCreator<AddPlanDefinitionAction>;
```

Comment:
`ActionCreator<Action>` is not type-safe as `typeof actionCreator`. You can try removing one of the args from any of the `ActionCreator<Action>`-typed action creators and you will see that no error is thrown(or atleast my editor does not show the red scriggly lines.)